packages/sync-service/lib/electric/tenant_manager.ex

```diff
@@ -187,8 +187,25 @@ defmodule Electric.TenantManager do
           connection_opts: connection_opts
         ]
 
+    # We deliberately don't include the electric instance ID
+    # in the publication and slot name because
+    # those are shared between all Electric instances connected to the same PG DB
+    # and there may be cases where we want one Electric to take over the publication
+    # and slot for a certain tenant of another Electric instance, e.g. during a rolling deploy.
+    # Note that this assumes that tenant IDs are unique across all Electric instances.
+    tenant_hash = tenant_id |> :erlang.phash2() |> Integer.to_string()
+
+    updated_app_config =
```

Comment:
```suggestion
    per_tenant_app_config =
```
maybe more semantically rich than "updated"?